example/example-html.coffee

```diff
@@ -0,0 +1,233 @@
+        "color": "purple",
+        "blocks": [
+          {"block":"<p>\\n  \\n</p>"},
+          {"block":"<hr />"},
+          {"block":"<pre>\\n  \\n</pre>"},
+          {"block":"<blockquote>\\n  \\n</blockquote>"},
+          {"block":"<ol>\\n  \\n</ol>"},
+          {"block":"<ul>\\n  \\n</ul>"},
+          {"block":"<li></li>"},
+          {"block":"<dl>\\n  \\n</dl>"},
+          {"block":"<dt></dt>"},
+          {"block":"<dd></dd>"},
+          {"block":"<figure>\\n  \\n</figure>"},
+          {"block":"<figcaption></figcaption>"},
+          {"block":"<main>\\n  \\n</main>"},
+          {"block":"<div>\\n  \\n</div>"}
+        ]
+      },
+      {
+        "name": "Text",
+        "color": "lightgreen",
+        "blocks": [
+          {"block":"<a href=\\"\\"></a>"},
+          {"block":"<em></em>"},
+          {"block":"<strong></strong>"},
+          {"block":"<small></small>"},
+          {"block":"<big></big>"},
+          {"block":"<cite></cite>"},
+          {"block":"<q></q>"},
+          {"block":"<dfn></dfn>"},
+          {"block":"<abbr title=\\"\\"></abbr>"},
+          {"block":"<ruby>\\n  \\n</ruby>"},
+          {"block":"<rt></rt>"},
+          {"block":"<rp></rp>"},
+          {"block":"<data></data>"},
+          {"block":"<data value=\\"\\"></data>"},
+          {"block":"<time></time>"},
+          {"block":"<code></code>"},
+          {"block":"<var></var>"},
+          {"block":"<samp></samp>"},
+          {"block":"<kbd></kbd>"},
+          {"block":"<sub></sub>"},
+          {"block":"<sup></sup>"},
+          {"block":"<i></i>"},
+          {"block":"<b></b>"},
+          {"block":"<u></u>"},
+          {"block":"<mark></mark>"},
+          {"block":"<bdi></bdi>"},
+          {"block":"<bdo dir=\\"\\">\\n  \\n</bdo>"},
+          {"block":"<span></span>"},
+          {"block":"<br />"},
+          {"block":"<wbr />"},
+          {"block":"text"}
+        ]
+      },
+      {
+        "name": "Other",
+        "color": "pink",
+        "blocks": [
+          {"block":"<ins></ins>"},
+          {"block":"<del></del>"},
+          {"block":"<details>\\n  \\n</details>"},
+          {"block":"<summary></summary>"},
+          {"block":"<menu>\\n  \\n</menu>"},
+          {"block":"<menuitem></menuitem>"},
+          {"block":"<dialog open></dialog>"},
+          {"block":"<script>\\n  \\n</script>"},
+          {"block":"<script src=\\"\\"></script>"},
+          {"block":"<noscript></noscript>"},
+          {"block":"<template>\\n  \\n</template>"},
+          {"block":"<canvas></canvas>"},
+          {"block":"<applet code=\\"\\">\\n  \\n</applet>"},
+          {"block":"<basefont color=\\"\\" size=\\"\\" />"},
+          {"block":"<bgsound src=\\"\\" />"},
+          {"block":"<center>\\n  \\n</center>"},
+          {"block":"<command type=\\"\\" label=\\"\\" />"},
+          {"block":"<font sixe=\\"\\" color=\\"\\">\\n  \\n</font>"},
+          {"block":"<frameset cols=\\"\\">\\n  \\n</frameset>"},
```

Comment:
This problem is only for tags which are not part of HTML5 and were initially not present in the palette.
Later I added the commonly used deprecated tags to the palette but couldn't find a proper documentation of where they can be present(because they aren't part of HTML5).
So I guess I'll just go with what you mentioned.